armi/bookkeeping/db/__init__.py

```diff
@@ -78,7 +78,13 @@
 ]
 
 
-def loadOperator(pathToDb, loadCycle, loadNode, allowMissing=False):
+def loadOperator(
+    pathToDb,
+    loadCycle,
+    loadNode,
+    statePointName=None,
+    allowMissing=False,
```

Comment:
I won't die on this hill, but we would have more backwards compatibility if the new method parameter (`statePointName`) came last.

Reply:
fair, done.